packages/terra-collapse/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "terra-collapse",
+  "main": "lib/Collapse.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Collapse component that transitions content in and out with the click on a button.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-collapse",
+    "Collapse",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
```

Comment:
Should be 15.4.2. Looks like this is coming from the [generator](https://github.com/cerner/generator-terra-module/blob/master/generators/app/templates/terra-package.json#L28-L29). I'll update this, need to check other packages as well and make sure there at 15.4.2.